python/ray/tune/tests/test_searchers.py

```diff
@@ -327,9 +327,13 @@ def testSkopt(self):
         self.assertCorrectExperimentOutput(out)
 
     def testZOOpt(self):
+        self.skipTest(
+            "Recent ZOOpt versions fail handling invalid values gracefully. "
+            "Skipping until a fix is added in a future ZOOpt release."
+        )
```

Comment:
To ensure we eventually re-enable this test, should we create an issue to track this?